calcite-adapter/src/main/java/software/amazon/documentdb/jdbc/calcite/adapter/DocumentDbRules.java

```diff
@@ -382,22 +383,43 @@ protected RexToMongoTranslator(final JavaTypeFactory typeFactory,
             }
 
             switch (literal.getType().getSqlTypeName()) {
-                case DOUBLE:
                 case DECIMAL:
```

Comment:
DECIMAL should be its own case [mapped](https://docs.mongodb.com/manual/reference/mongodb-extended-json/#mongodb-bsontype-Decimal128) to `$numberDecimal` instead of `$numberDouble`. Is there a separate case for NUMERIC? Or is that just DECIMAL?

Reply:
DocumentDB doesn't support the 128 bit decimal type 
And yup! Numeric = Decimal.

Reply:
Added TODO